### DISCUSSION TITLE
Real Device Native App/Browser support or Appium integration

### DISCUSSION THREAD

**Discussion by sridharaiyer** [seleniumbase/SeleniumBase#816] [Ideas]
Hi Seleniumbase Team.

Are there any plans to support real device Native Apps or Device Browser automation or Appium integration?

**Comment by mdmintz** [comment]
Hello @sridharaiyer See SeleniumBase mobile mode, which lets you simulate any mobile device in a web browser: https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/mobile_testing.md

**Reply by Jeremy-QA** [reply]
@sridharaiyer I’ve already integrated Appium and SeleniumBase. Now, just download all the necessary Appium dependencies and import them into your project. Use pytest to manage the interaction between Appium and SeleniumBase
